2022-08-08 15:49:02,553 [app] [ERROR] :40: Exception on /vod_detail [GET]
Traceback (most recent call last):
  File "D:\Demo\flask-comic\venv\lib\site-packages\flask\app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Demo\flask-comic\venv\lib\site-packages\flask\app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Demo\flask-comic\venv\lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Demo\flask-comic\venv\lib\site-packages\flask\app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Demo\flask-comic\venv\lib\site-packages\flask\app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "D:\Demo\flask-comic\comic_sakura\blueprints\sakura_vod_info.py", line 86, in get_vod_detail
    for play_url_set in mov['vod_play_url'].split('#'):
TypeError: 'MovDetail' object is not subscriptable
